invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -41,9 +41,9 @@
         {% set is_community_restricted = community.access.visibility == 'restricted' %}
         <div class="ui fluid container page-subheader-outer with-submenu compact">
           <div class="ui container page-subheader">
-            <div class="ui mini image community-logo page-subheader-element ">
+            <div class="page-subheader-element ">
               <img
-                class=""
+                class="ui tiny image community-header-logo"
```

Comment:
if I remember correctly, the issue was also present in the deposit form, is there a PR fixing it?

Reply:
it is correct in the deposit form but i suspect you maybe did not have the latest one/not released yet?

Reply:
that could be the case, I will reinstall. Thanks for checking